sonic_package_manager/main.py

```diff
@@ -378,17 +642,50 @@ def install(ctx,
     if allow_downgrade is not None:
         install_opts['allow_downgrade'] = allow_downgrade
 
+    if use_local_manifest:
```

Comment:
For now, I have made this option(use_local_manifest) to be hidden.  Have kept this code as such in case if there is a requirement in future to override the built-in manifest.